naucse/markdown_util.py

```diff
@@ -67,6 +67,12 @@ def ansi_convert(code):
     return ansi_convertor.convert(replaced, full=False)
 
 
+def style_space_after_prompt(html):
+    return re.sub(r'<span class="gp">([^<]+)</span>(\s+)',
+                  r'<span class="gp">\1</span><span class="gp_s">\2</span>',
```

Comment:
Nitpick – I don't see the purpose of `gp_s`. Did you think about including the space in the `gp` span?

Reply:
I wanted to keep `gp` unchanged, not to break other (possibly future) usecases.

Reply:
Apparently, some gps end with whitespace, so I'll do it as @encukou suggested.